bluebottle/bb_accounts/models.py

```diff
@@ -292,8 +292,7 @@ def task_count(self):
         task member in
         """
         task_count = Task.objects.filter(author=self).count()
-        taskmember_count = TaskMember.objects.filter(
-            member=self, status__in=['applied', 'accepted', 'realized']).count()
```

Comment:
With this `task_count` will also count rejected & withdrawn?

Reply:
Yes. It should count all task members.